### DISCUSSION TITLE
Execute middleware before authChecker

### DISCUSSION THREAD

**Discussion by ericlammers** [MichalLytek/type-graphql#1002] [Q&A] [answered]
I would like to execute middleware before the authChecker runs. Is this possible?

**Comment by MichalLytek** [comment, marked as answer]
Nope.

**Reply by ericlammers** [reply]
Alright. Thanks for confirming. Love the library.

**Reply by MichalLytek** [reply]
You can write your own auth middleware and run that after the main middleware.
